connectors/utils.py

```diff
@@ -399,3 +401,74 @@ async def func_to_execute(*args, **kwargs):
         return func_to_execute
 
     return wrapper
+
+
+def ssl_context(certificate):
```

Comment:
Moving these methods and adding new ones should be separate PR. Let's focus this one only on `ssl_disabled` -> `ssl_enabled`

Reply:
Removed the unrelated changes form the PR. You can take a look.